server/autotest_server/testers/py/py_tester.py

```diff
@@ -211,11 +227,11 @@ def __init__(
 
         The result was created after running some unittest or pytest tests.
         """
-        self._test_name = result["name"]
```

Comment:
please revert these changes (it's just reordering, with no other changes now that you removed the attribute)

Reply:
Done.